src/03-modality-agnostic-files.md

```diff
@@ -143,8 +143,16 @@ Example:
 
 ### `README`
 
-Every BIDS dataset SHOULD come with a free form text file (`README`) describing the dataset in more detail.
-The `README` file MUST be either in ASCII or UTF-8 encoding.
+A REQUIRED free-form text file, `README`, SHOULD describe the dataset in more detail.
+The `README` file MUST be either in ASCII or UTF-8 encoding and MAY have one of the extensions:
+`.md` ([Markdown](https://www.markdownguide.org/)),
+`.rst` ([reStructuredText](https://docutils.sourceforge.io/rst.html)),
+or `.txt` (unstructured text).
+BIDS does not make any recommendations with regards to the
+[Markdown flavor](https://www.markdownguide.org/extended-syntax/#lightweight-markup-languages)
```

Comment:
I think the goal was not to validate the syntax. Therefore, no recommendation for the flavor. In my opinion basic would be too restrictive. For example tables and task lists are useful but not included in Basic syntax.

Reply:
> I think the goal was not to validate the syntax.

indeed, this has been discussed in several threads and in two (or so) online meetings, notes can be found here:

- https://github.com/bids-standard/bids-validator/pull/1281#discussion_r630118909
- https://github.com/bids-standard/bids-specification/issues/972
- Maintainer Biweekly meeting notes (links to a specific comment therein): https://docs.google.com/document/d/1MuwZt6ToZmsPgH6naFVlPDFtNUz61DzlaZmdNHIAotg/edit?disco=AAAAUZdMt60

> In my opinion basic would be too restrictive. 

agreed. Regarding your concern:

1. People will write their README.md file with flavors such that it renders **at least**  on the platform(s) of their choice. For all other platforms, our recommendation kicks in: READMEs SHOULD be understandable/parsable even in unrendered form
2. READMEs will most likely not include very very advanced MD syntax, and most flavors have significant overlap, which furthermore reduces the problem

I therefore stand by the current proposal to not recommend any flavor.

Reply:
1. many people already use MD and have to put their README in .bidsignore to make their dataset valid
2. other people add a txt extension because they are used to having extensions, and are then surprised that this does not work
3. as I mention above: the currently proposed solution will work well for almost all cases: Those cases where people use basic markdown syntax and perhaps a markdown table ... for cases where the syntax cannot be properly rendered due to a flavor incompatibility between README and rendering host, we still have ensured the readability of the README (albeit slightly more ugly; but note: this will happen rarely)

Overall I believe that the current solution (no flavor recommendation, no validation) is a good tradeoff between utility gains for users and validation/specification burden for BIDS. 

> even in the minority I am, I still feel wrong to give up on this

Please feel free to tag more people if you want to discuss this more broadly -- I hope that we can find a consensus.